src/compiler/reviews.md

```diff
@@ -1,4 +1,159 @@
-# Review policies
+Review Policy
+=============
+
+It is the purpose of code reviews to
+
+- **reduce** the risk of introducing **bugs** and usability and performance **regressions**,
+- keep our code **maintainable**, **readable**, and **documented**, and
+- ensure that changes are made with the big picture and **appropriate context in mind**.
+
+Reviewing accomplishes this by bringing in another set of eyes
+to look at a proposed change from a different perspective,
+which increases the chance of catching mistakes early and
+uncovering potential blind spots in the reasoning behind the change.
+
+
+## Basic Reviewing Requirements
+
+There are a number of requirements that need to be met in order for reviewing to be effective:
+
+- Reviewers must have a sufficient **understanding of the code** under review.
+    > This is important to help spot non-obvious, unintentional side effects of a given change.
+- Pull requests must provide (1) a concise high-level **description of the change** and (2) the **rationale** behind it.
```

Comment:
Nit: do we want to rephrase this in terms of what the PR author must provide? The other points are about obligations of the reviewer, but this point is about what a pull request must provide. Obviously, pull requests don't have the ability to provide anything (they're just UI on GitHub) - it is the responsibility of the author to provide these things *in* a pull request description.

Reply:
Done.